web/src/pages/your_account.rs

```diff
@@ -1,49 +1,140 @@
-use std::collections::HashMap;
+use std::{borrow::Cow, collections::HashMap, fmt, mem};
 
-use at2_ns::ThinUser;
+use at2_ns::{FullUser, ThinUser};
 use js_sys::{JsString, Reflect};
-use yew::{prelude::*, services::ConsoleService, worker::Agent};
+use material_yew::{
+    dialog::{ActionType, MatDialogAction},
+    MatButton, MatDialog, MatFormfield, MatList, MatListItem, WeakComponentLink,
+};
+use yew::{prelude::*, worker::Agent};
 
-use crate::users_agent::UsersAgent;
+use crate::agents;
+
+const DEFAULT_SEND_TRANSACTION_AMOUNT: usize = 12;
+
+// TODO snafu?
+#[derive(Debug)]
+struct MissingField;
+
+impl fmt::Display for MissingField {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.write_str("missing field")
+    }
+}
+impl std::error::Error for MissingField {}
+
+// TODO split
```

Comment:
are these TODOs for later?

Reply:
Good catch, it was to be handled before the PR, fixing it now :)